profiles/scraper.py

```diff
@@ -522,7 +573,13 @@ def parse_profile_education(self):
         :return: List of dictionaries containing information about each education. Return [] if above condition does not satisfy.
         """
         educations = []
-        get_educations = self.get_multiple_by_xpath_or_none("EDUCATION_SINGLE")
+        get_educations = self.get_multiple_by_xpath_or_none("EDUCATION_MULTIPLE")
+        if not get_educations: # if there isn't multiple educations, but this person has 1 education
+            single_education_web_element = self.get_by_xpath_or_none("EDUCATION_SINGLE")
+            if not single_education_web_element:
+                return None
+            get_educations = []
+            get_educations.append(single_education_web_element)
```

Comment:
EDUCATION_SINGLE works because it calls get_multiple_by_xpath_or_none so it returns a list